packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -11,101 +11,19 @@ import 'package:path/path.dart' as path;
 import 'package:yaml/yaml.dart' as yaml;
 
 import '../artifacts.dart';
-import '../base/logger.dart';
-import '../base/process.dart';
 import '../base/utils.dart';
 import '../build_configuration.dart';
+import '../dart/analysis.dart';
 import '../dart/sdk.dart';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
 
+
 bool isDartFile(FileSystemEntity entry) => entry is File && entry.path.endsWith('.dart');
```

Comment:
Are these still used? You might try making them private to see if they're still in use.